wordpress-mobile/WordPress-iOS
# Reader: Add new reader post card cell UI

Part of #21645
Merges into #21683 

## Description

Adds the new Reader card UI with filler content. I'm breaking up the PR's due to the large number of changes.

> **Note**
> The following will be in future PR's:
> - Configuring the view with a reader post
> - Button actions
> - Safe area fixes
> - Accessibility fixes

## Screenshots

| Light | Dark |
|------|------|
| ![light](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/b089e245-833f-444a-b4c4-668af6a2cf65) | ![dark](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/583dbd4d-3610-466e-8f77-5544aa6ad76c) |

## Testing

To test:
- Launch Jetpack and Login
- Enable the `Reader Improvements v1` feature flag if necessary
- Tap on Reader
- 🔎 **Verify**:
  - Layout matches the spec for the reader card zQnohyMpLzBzQ5jzMMKni3-fi-868_19180
  - There are no breaking constraints, including when rotating the device

## Regression Notes
1. Potential unintended areas of impact
Old reader post card cell

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
